lib/itamae/plugin/recipe/docker/install.rb

```diff
@@ -75,7 +75,7 @@
   end
 
 else
-  require 'itamae/plugin/recipe/version'
+  require_relative './version'
```

Comment:
Actually this was typo of `require 'itamae/plugin/recipe/docker/version'`. But I'm okay with this.